vm/policy.go

```diff
@@ -38,6 +42,54 @@ func PolicyHandling(t string, path string, o PolicyOptions) error {
 		return err
 	}
 
+	client := pb.NewPolicyServiceClient(conn)
+
+	req := pb.Policy{
+		Policy: policyEventData,
+	}
+
+	if resp, err := client.HostPolicy(context.Background(), &req); err == nil {
+		if resp.Status == 1 {
+			fmt.Print("Success")
+		} else {
+			return fmt.Errorf("failed to send policy")
+		}
+	}
+
+	return nil
```

Comment:
```suggestion
	resp, err := client.HostPolicy(context.Background(), &req)
	if err != nil || resp.Status !=1 {
		return fmt.Errorf("failed to send policy %v",err)
	}
	fmt.Print("Success")
	return nil
```
We don't consider the case where err is not nil

Reply:
Need to discuss on the same